### DISCUSSION TITLE
Hacktoberfest Banner

### DISCUSSION THREAD

**Discussion by soumikbaithalu** [Tenet-Coding/Hacktoberfest-Projects#9] [Ideas]
Which banner is best ?? 
1. 
![banner08CRredlogo](https://user-images.githubusercontent.com/66599363/134625438-13b40dd6-9068-4f9e-aefa-1bcb96ffb10a.png)

2.
![footer-simple-light](https://user-images.githubusercontent.com/66599363/134625595-41617410-da0b-48fd-922f-ea69989780b4.png)

**Comment by Harshil-Jani** [comment]
1st one
